database.tf

```diff
@@ -4,7 +4,7 @@ locals {
     business   = ["1", "2"],
     entreprise = ["1", "2", "3"]
   }
-  nodes_set = lookup(local.nodes_iterator, var.database_plan)
```

Comment:
is it linked to ovh provider version bump?

Reply:
no, but tflint did not let the CI pass because of this deprecated use of lookup function.
So I had to modify it in order not to be blocked